### DISCUSSION TITLE
Sync meta data of imported test cases

### DISCUSSION THREAD

**Discussion by bendrissou** [AFLplusplus/AFLplusplus#2162] [General]
Hi,

When importing test cases of other parallel nodes, the original timestamp in the file name is lost. The new test case file name is constructed as follows:

```
sprintf(ret, "sync:%s,src:%06u", afl->syncing_party, afl->syncing_case);
```

Does AFL store the file name (or other meta data) of the imported test case somewhere?

Thank you.

**Comment by vanhauser-thc** [comment]
You see the fuzzer name that it synced from and the id it has there. Just look it up there

**Comment by bendrissou** [comment]
Yes I am aware that I have the fuzzer name and the id. So, I can write a script to look up other information.

So you confirm that the full filename is not stored in the state. And can't be retrieved in AFL.

I am most interested in the timestamp of the imported file.
